tools/setup-protobuf.sh

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 pushd /tmp
-curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v21.12/protoc-21.12-linux-x86_64.zip
-unzip -o protoc-21.12-linux-x86_64.zip -d /usr/local/
+curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.14.0/protoc-3.14.0-linux-x86_64.zip
+unzip -o protoc-3.14.0-linux-x86_64.zip -d /usr/local/
```

Comment:
Why do we downgrade protobuf version? should we use the same version as it in `ci/install_protobuf.sh` ?

Reply:
@owent 

This commit is actually a revert, to leave the `tools/setup-protobuf.sh` file unchanged compared to the main branch.

`workflows/ci.yml` uses `./ci/install_protobuf.sh`